src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts

```diff
@@ -32,83 +32,65 @@ export interface FileProperties {
     selector: 'jhi-attachment-unit-form',
     templateUrl: './attachment-unit-form.component.html',
 })
-export class AttachmentUnitFormComponent implements OnInit, OnChanges {
-    @Input()
-    formData: AttachmentUnitFormData;
-    @Input()
-    isEditMode = false;
+export class AttachmentUnitFormComponent implements OnChanges {
+    protected readonly faQuestionCircle = faQuestionCircle;
+    protected readonly faTimes = faTimes;
+    protected readonly allowedFileExtensions = ALLOWED_FILE_EXTENSIONS_HUMAN_READABLE;
+    protected readonly acceptedFileExtensionsFileBrowser = ACCEPTED_FILE_EXTENSIONS_FILE_BROWSER;
 
-    // A human-readable list of allowed file extensions
-    readonly allowedFileExtensions = UPLOAD_FILE_EXTENSIONS.join(', ');
-    // The list of file extensions for the "accept" attribute of the file input field
-    readonly acceptedFileExtensionsFileBrowser = UPLOAD_FILE_EXTENSIONS.map((ext) => '.' + ext).join(',');
+    @Input() formData: AttachmentUnitFormData;
+    @Input() isEditMode = false;
 
-    faQuestionCircle = faQuestionCircle;
+    @Output() formSubmitted: EventEmitter<AttachmentUnitFormData> = new EventEmitter<AttachmentUnitFormData>();
 
-    @Output()
-    formSubmitted: EventEmitter<AttachmentUnitFormData> = new EventEmitter<AttachmentUnitFormData>();
-    form: FormGroup;
-
-    @Input()
-    hasCancelButton: boolean;
-    @Output()
-    onCancel: EventEmitter<any> = new EventEmitter<any>();
-
-    faTimes = faTimes;
+    @Input() hasCancelButton: boolean;
+    @Output() onCancel: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
The diff is rather hard to read here, only the order and formatting of the Iputs and Outputs are changed as we use to write them in one line